cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -344,18 +304,18 @@ func getOrCreateCStorVolumeResource(
 // on the available cstor pools created for storagepoolclaim.
 // if pools are less then desired replicaCount its return an error.
 func distributeCVRs(
-	replicaCount int,
+	pendingReplicaCount int,
+	claim *apis.CStorVolumeClaim,
 	service *corev1.Service,
 	volume *apis.CStorVolume,
-	class *storagev1.StorageClass,
 ) error {
 
-	spcName := getSPC(class)
+	spcName := getSPC(claim)
```

Comment:
you want to say it as getCSPC, cspcName in the entire PR?

Reply:
this can be taken up once all the CSPC PRs are merged.. for this PR, lets go ahead with SPC

Reply:
Done